bin/start-openshift-docker.sh

```diff
@@ -192,9 +195,10 @@ getServiceIp()
 }
 
 FABRIC8_CONSOLE=http://$(getServiceIp "$K8S_SERVICES" fabric8-console)/
-DOCKER_REGISTRY=http://$(getServiceIpAndPort "$K8S_SERVICES" registry)
+DOCKER_REGISTRY=$(getServiceIpAndPort "$K8S_SERVICES" registry)
```

Comment:
Sorry, this fixes a bug in the script and isn’t related to my other changes. Exporting DOCKER_REGISTRY=http://… causes 'docker push’ to fail.

Doug

> On Mar 2, 2015, at 11:30, Jimmi Dyson <notifications@github.com> wrote:
> 
> In bin/start-openshift-docker.sh <https://github.com/fabric8io/fabric8/pull/3517#discussion_r25629483>:
> 
> > @@ -192,9 +195,10 @@ getServiceIp()
> >  }
> >  
> >  FABRIC8_CONSOLE=http://$(getServiceIp "$K8S_SERVICES" fabric8-console)/
> > -DOCKER_REGISTRY=http://$(getServiceIpAndPort "$K8S_SERVICES" registry)
> > +DOCKER_REGISTRY=$(getServiceIpAndPort "$K8S_SERVICES" registry)
> Why this line change?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/fabric8io/fabric8/pull/3517/files#r25629483>.
>